specification/deviceregistry/resource-manager/Microsoft.DeviceRegistry/preview/2023-11-01-preview/asset.json

```diff
@@ -0,0 +1,663 @@
+          }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "assetNameParameter": {
+      "name": "assetName",
+      "description": "Asset name parameter.",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "pattern": "^[a-z0-9][a-z0-9-]*[a-z0-9]$",
+      "maxLength": 63,
+      "minLength": 3
+    }
+  },
+  "definitions": {
+    "AssetResponseList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Asset"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "type": "string"
+        }
+      },
+      "description": "List of Assets."
+    },
+    "Asset": {
+      "type": "object",
+      "required": [
+        "extendedLocation"
+      ],
+      "properties": {
+        "properties": {
+          "type": "object",
+          "required": [
+            "assetEndpointProfileUri"
+          ],
+          "allOf": [
+            {
+              "$ref": "#/definitions/AssetProperties"
+            }
+          ],
+          "description": "Asset resource properties."
+        },
+        "extendedLocation": {
+          "type": "object",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ],
+          "properties": {
+            "type": {
+              "type": "string",
+              "description": "The extended location type."
+            },
+            "name": {
+              "type": "string",
+              "description": "The extended location name."
+            }
+          },
+          "description": "The extended location."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "Asset definition."
+    },
+    "AssetPatchPayload": {
+      "type": "object",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "x-ms-mutability": [
+            "read",
+            "create",
+            "update"
+          ],
+          "description": "Resource tags."
+        },
+        "properties": {
+          "$ref": "#/definitions/AssetProperties",
+          "description": "Asset resource properties."
+        }
+      },
+      "description": "Asset definition for Patch operation."
+    },
+    "AssetProperties": {
+      "description": "Defines the asset properties.",
+      "type": "object",
+      "properties": {
+        "uuid": {
+          "type": "string",
+          "description": "Globally unique, immutable, non-reusable id.",
+          "readOnly": true
+        },
+        "assetType": {
+          "type": "string",
+          "description": "Resource path to asset type (model) definition."
+        },
+        "enabled": {
```

Comment:
ARM API best practices suggest converting booleans to enums. See [ARM API Review best-practices](https://eng.ms/docs/products/arm/rp_onboarding/process/api_review_best_practices) for details.

Reply:
We don't expect to have any different values other than `true` or `false` here in the future.